packages/pv-scripts/scripts/prod.js

```diff
@@ -46,6 +49,21 @@ function webpackBuild(webpackConfig) {
         return reject(new Error(messages.errors.join("\n\n")));
       }
 
+      // generate the css stats
+      if (process.env.PV_WEBPACK_STATS) {
+        const destPath = getAppConfig().destPath;
+        explore(`${path.resolve(".", destPath, "css")}/*.css`, {
+          output: {
+            format: process.env.PV_WEBPACK_STATS,
+            filename: `${path.resolve(".", "target/report_css")}.${process.env.PV_WEBPACK_STATS}`,
```

Comment:
hardcoded target/ ? we should use configured destFolder from pv.config.js

Reply:
good catch! used the destPath for reading the css files but not for writing the output.